src/config/Config.zig

```diff
@@ -2732,8 +2733,8 @@ pub fn loadRecursiveFiles(self: *Config, alloc_gpa: Allocator) !void {
                 const diag: cli.Diagnostic = .{
                     .message = try std.fmt.allocPrintZ(
                         arena_alloc,
-                        "error opening config-file {s}: {}",
-                        .{ path, err },
+                        "{s} config-file {s}: {}",
```

Comment:
Not everyone understands what `config-file` means, especially when we implement GUI preferences and users won't be able to make the connection between the name and the config setting